misk/src/test/kotlin/misk/client/MockWebServerService.kt

```diff
@@ -6,22 +6,22 @@ import java.io.File
 import javax.inject.Singleton
 
 @Singleton
-class MockWebServerService(val socketFile: String?) : AbstractIdleService() {
+class MockWebServerService(val unixSocketFile: String?) : AbstractIdleService() {
```

Comment:
I actually meant we should rename the class to make it clear it uses a socket file. although that would be a total mouthful of a name, so whatever.

Reply:
well the idea is that the socket file specification is optional. You can run this as a mock server without specifying a unix socket.